src/decay.cpp

```diff
@@ -107,10 +107,10 @@ const Decay& Decays_list::get_decay(
 
       std::size_t count = 0;
       const std::size_t n_final = product_pdgs.size();
-      for (auto it = std::begin(product_pdgs),
-              end = std::end(product_pdgs); it != end; ++it, ++count) {
-         msg << *it;
-         if (count != n_final) {
+      for (int pdg : product_pdgs) {
+         count++;
+         msg << pdg;
+         if (++count != n_final) {
```

Comment:
Isn't there one `++count` too much?

Reply:
Ah, yes! Sorry about that.

Reply:
I've just added an overload function `concat()` (commit 3562dc10f), which you can use replace this loop by a call of the form:
~~~.cpp
std::string msg = concat(product_pdgs.cbegin(), product_pdgs.cend(), ',');
~~~

Reply:
I'm not sure how helpful it is. What I have is a list of integers. I'd first have to convert it to list of `strings` and then call `concat`. This will probably make the code longer.

Reply:
Well, the function that I've added will properly treat ints, see the example:
https://github.com/FlexibleSUSY/FlexibleSUSY/blob/development/test/test_string_utils.cpp#L42
So, I guess it'll make the code shorter. :)

Reply:
Done in c1e3a301338a20300aed00ee0e8ae6762e10b1d5.